charts/aws-mountpoint-s3-csi-driver/templates/node.yaml

```diff
@@ -37,6 +37,9 @@ spec:
       initContainers:
         - name: install-mountpoint
           image: {{ printf "%s%s:%s" (default "" .Values.image.containerRegistry) .Values.image.repository (default (printf "v%s" .Chart.AppVersion) (toString .Values.image.tag)) }}
+          {{- if .Values.imagePullSecrets }}
+          imagePullSecrets: {{- toYaml .Values.imagePullSecrets | nindent 8 }}
```

Comment:
since we know it's a list, and should contain a `name` field, thoughts on doing it like this?
```
      {{- if .Values.imagePullSecrets }}
      imagePullSecrets:
      {{- range .Values.imagePullSecrets }}
        - name: {{ . }}
      {{- end }}
      {{- end }}
```

Reply:
@truestory1 take a look at this comment, still holds with the recent changes! this way users don't have to append `name:` at the beginning of each value

Reply:
With recent change we have:

```
$ helm template -s templates/node.yaml . | yq '.spec.template.spec' | head -15                                                        

nodeSelector:
  kubernetes.io/os: linux
serviceAccountName: s3-csi-driver-sa
priorityClassName: system-node-critical
tolerations:
  - key: CriticalAddonsOnly
    operator: Exists
  - operator: Exists
    effect: NoExecute
    tolerationSeconds: 300
imagePullSecrets:
  - name: test-secret-name
initContainers:
  - name: install-mountpoint
    image: public.ecr.aws/mountpoint-s3-csi-driver/aws-mountpoint-s3-csi-driver:v1.1.0

```
where
```
$ cat values.yaml | grep -A1 imagePullSecrets                                                                                         
imagePullSecrets:
  - test-secret-name
```

PS: I was confused by using or non-using the name field. Thanks again for pointing me to right destination